### ISSUE TITLE
Input of KLDivLoss

### ISSUE THREAD

**Issue by pvmilk** [RerRayne/bc_learning_pytorch#2]
I believed that the input of KLDivLoss should be log-probability. 
https://pytorch.org/docs/stable/nn.html?highlight=kldivloss#torch.nn.KLDivLoss

In the code, the output of the model is the softmax. 
I think it is suppose to be LogSoftmax() at the last layer.
